docs/reference/stores.mdx

```diff
@@ -56,4 +56,67 @@ again have one `StoreBuilder` (with a unique `StoreSupplier`) for each state sto
             new StringSerde()
         )
     );
+```
+
+## Responsive StateStore Features
+
+Responsive state stores offer some features beyond what's available for the built-in stores of Kafka Streams:
+
+### Time-to-live for key-value stores
+
+Key-value stores in Kafka Streams have infinite retention, but this can be undesirable for data sets with an
+unbounded keyspace or timeseries data. To address these use cases and minimize storage needs, Responsive allows you
+to define a ttl (time-to-live) on any KeyValueStore to set a retention time for the data inside it.
+
+Responsive ttl is most effective for non-historical data, as the data is actively expired based on wallclock time
```

Comment:
let's move this to an asterisk or something at the bottom, it's pretty confusing for people who haven't thought this through or know what 'historical' data means.